ScarletEngine/Sources/AchillesPhysics/Private/ColliderSystem.cpp

```diff
@@ -0,0 +1,114 @@
+#include "ColliderSystem.h"
+#include "RigidBodyComponent.h"
+#include "Core.h"
+
+namespace ScarletEngine::Achilles
+{
+	void AABBvsAABBColliderSystem::Update() const
+	{
+	}
+
+	void AABBvsAABBColliderSystem::FixedUpdate() const
+	{
+		// Move collider
+		for (const auto& [Entity, Box, Rb] : GetEntities<BoxColliderComponent, RigidBodyComponent>())
+		{
+			glm::vec3 Dx = Rb->Velocity * (float)FIXED_UPDATE_S;
+			Box->Max += Dx;
+			Box->Min += Dx;
+		}
+
+		// @todo : bounce
+	}
+
+	void SphereVsSphereColliderSystem::Update() const
+	{
+	}
+
+	void SphereVsSphereColliderSystem::FixedUpdate() const
+	{
+		// Move collider
+		for (const auto& [Entity, Rb, Sphere] : GetEntities<RigidBodyComponent, SphereColliderComponent>())
+		{
+			Sphere->Pos += Rb->Velocity * (float)FIXED_UPDATE_S;
+		}
+
+		// Compute intersection
+		const auto& Entities = GetEntities<Transform, SphereColliderComponent, RigidBodyComponent>();
+		int size = Entities.size();
```

Comment:
`size` doesn't follow naming convention. Variables should begin with an uppercase letter.

In addition, size can be made const here.

Finally, the value returned from `Entities.size()` is of type `size_t`. Prefer using that here over `int`.